propertyestimator/protocols/paprika.py

```diff
@@ -513,7 +519,7 @@ def _perform_analysis(self, directory):
                 self._results_dictionary['release']['ti-block']['sem'] * unit.kilocalorie / unit.mole,
                 self._id + "_release")
 
-        if 'ref_state_work' in self._results_dictionary:
+        if 'ref_state_work' in self._results_dictiohost_protocol.schemanary:
```

Comment:
I think this is a typo?

Reply:
Yes it is a typo. Thanks for catching it.